data-pipeline/schemas/gnomad_v4_variants/annotate_gnomad_v4_transcript_consequences/mane_transcripts_path/mane_select_transcripts.ht.schema

```diff
@@ -0,0 +1,14 @@
+----------------------------------------
```

Comment:
The reasoning behind the schemas are described in the (overly long) PR description above:

> This is analogous to our jest snapshots. If anything in the pipeline changes, the schema will be tracked in source control. And it's nice to have as a reference.

It turned out to be actually quite useful for v4 development. When I'd get an update table from methods team, after running the pipeline the diff would show how each step of the pipeline changed for all inputs/outputs. 

When my final HT had no schema changes I was able to confidently proceed to loading the database without any manual schema checks.

I'd like to expand this workflow to all of the pipelines, but this will require some refinement of the workflow. For example, currently I have to manually copy the `schemas` folder to my local machine using `scp`. Ideally this would happen as part of the submit script. Also it requires all pipelines to have a name. 

Anyways it is a non-breaking change so I think worth keeping.

Reply:
Here's an example commit, where git tracking caught some new changes to schemas:

https://github.com/broadinstitute/gnomad-browser/pull/1178/commits/38b94c0d209774ed00756264569922b7a44cb12a#diff-d830d725f3b5ba4c154e856461b71cf4a3d8524814c2dbe7ef2d788d53971a5e